usr/src/uts/common/fs/zfs/dmu_send.c

```diff
@@ -392,6 +392,22 @@ static int
 dump_freeobjects(dmu_sendarg_t *dsp, uint64_t firstobj, uint64_t numobjs)
 {
 	struct drr_freeobjects *drrfo = &(dsp->dsa_drr->drr_u.drr_freeobjects);
+	uint64_t maxobj = DNODES_PER_BLOCK *
+	    (DMU_META_DNODE(dsp->dsa_os)->dn_maxblkid + 1);
+
+	/*
+	 * Older versions of OpenZFS (since 7104) does not handle large 
+         * FREEOBJECTS records correctly, leading to zfs recv never completing. 
```

Comment:
Thanks for the update, @waikontse.

It looks like the indentation here is inconsistent.  Each line should start with `tab space asterisk`.

I think that in this context "since" means "more recent than", but I think you mean "less recent than" / "older than"?

Reply:
Hi @ahrens, thanks for the guidance on the commenting.
I need to get used to tab spaces, I usually use spaces. 

You're right, what I mean is that versions "less recent"/"older than".  I'll update the formatting with the comment later today.